### PR TITLE
[CU-357bfmr] Authentication API

### PR DESCRIPTION
**Pull request by sirateek**
Add `/auth/login/`

### CODE REVIEW

**Comment by sirateek** [comment]
Task linked: [CU-357bfmr Authenticate API](https://app.clickup.com/t/357bfmr)

**Comment by sirateek** [comment]
Wait for #6 to be merged

File: repository/population.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		return userInfo, err
+	}
+	userInfoLenght := len(userInfoList)
+	if userInfoLenght == 0 {
+		return userInfo, ErrNotFound
+	} else if userInfoLenght > 1 {
+		return userInfo, ErrFoundMoreThanOne
+	}
+	return *userInfoList[0], nil
+}
```
Comment (TaninDean) [inline repository/population.go:53]: It could user ```.Get``` same as my case or not. What do you think ?

Comment (sirateek) [reply repository/population.go]: Sure krub 👍

**Review by TaninDean — APPROVED** [review]

File: repository/population.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+func NewPopulationRepostory(mysql *sqlx.DB) PopulationRepository {
+	return &populationRepository{
+		mysql: mysql,
+	}
+}
+
+func (p *populationRepository) GetUserInfo(citizenID string) (userinfo model.UserInfo, err error) {
+	var getUserInfoList []model.UserInfo
+	err = p.mysql.Select(&getUserInfoList, "SELECT * from `Population` WHERE citizenID=?", citizenID)
```
Comment (abounk) [inline repository/population.go:26]: Should we also use ```.Get``` here?

Comment (sirateek) [reply repository/population.go]: Fixed krub

**Review by abounk — APPROVED** [review]
🤌🤌🤌

**Review by nabhan-au — APPROVED** [review]

**Pull request merged**